mmv1/templates/terraform/sweeper_file.go.tmpl

```diff
@@ -69,13 +98,23 @@ func testSweep{{ $.ResourceName }}(_ string) error {
 		t := &testing.T{}
 		billingId := envvar.GetTestBillingAccountFromEnv(t)
 
+		// Set fallback values for empty region/zone
+		if sub.region == "" {
+			log.Printf("[INFO][SWEEPER_LOG] Empty region provided, falling back to us-central1")
+			sub.region = "us-central1"
+		}
+		if sub.zone == "" {
+			log.Printf("[INFO][SWEEPER_LOG] Empty zone provided, falling back to us-central1-a")
+			sub.zone = "us-central1-a"
```

Comment:
It looks like previously this defaulted to `-` - just wanted to double-check that we want to change this?

Reply:
yeah I don't see a reason not to. I'm not sure where '-' is a valid value, but maybe we'll find out after the change gets in 'if any'.

Reply:
I don't know for sure, but I'm fine with moving forward with the change & finding out that way.